.github/workflows/dependabot_auto_merge.yml

```diff
@@ -15,19 +15,26 @@ jobs:
         uses: dependabot/fetch-metadata@v1
         with:
           github-token: '${{ secrets.GITHUB_TOKEN }}'
-      - name: Create changeset if not dev dependency
+
+      - name: Checkout (not dev dep)
+        uses: actions/checkout@master
+        if: ${{ steps.metadata.outputs.update-type != 'version-update:semver-major' && steps.metadata.outputs.dependency-type != 'direct:development' }}
```

Comment:
Can we not checkout and create in the same step? It feels strange to have to repeat the `if`. If it gets messy disregard this comment.